### DISCUSSION TITLE
Docker Networks

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#827] [Questions] [answered]
Why are the nextcloud-aio containers in the nextcloud-aio network, as well as in the docker bridge network?

**Comment by szaimen** [comment, marked as answer]
This is nothing we are doing manually. It seems to get applied by the docker daemon automatically.
We only assign them to the nextcloud-aio network. See https://github.com/nextcloud/all-in-one/blob/69382b638af5337f9b92ec3f5d634650a5988fb1/php/src/Docker/DockerActionManager.php#L568

I also tried to disconnect them from the bridge network but they seem to stop working then...
